packages/experiments/src/components/Chiclet/ChicletCard.tsx

```diff
@@ -0,0 +1,148 @@
+    const role = actionable ? (onClick ? 'button' : 'link') : undefined;
+    const tabIndex = actionable ? 0 : undefined;
+
+    var preview = this._renderPreviewImage(image, imageHeight, imageWidth, ogType);
+
+    return (
+      <div
+        tabIndex={ tabIndex }
+        role={ role }
+        onKeyDown={ actionable ? this._onKeyDown : undefined }
+        onClick={ actionable ? this._onClick : undefined }
+        className={
+          css('ms-ChicletCard', className) }
+      >
+        <div
+          className={ mergeStyles(this._classNames.preview) }
+        >
+          { image ?
+            preview :
+            (<Image
+              src={ TestImages.documentPreviewTwo }
+              role='presentation'
+              alt=''
+            />)
+          }
+        </div>
+        <div
+          className={ mergeStyles(this._classNames.info) }
+        >
+          <div
+            className={ mergeStyles(this._classNames.title) }
+          >
+            { title ? title : "Placeholder" }
+          </div>
+          <div
+            className={ mergeStyles(this._classNames.link) }
+          >
+            { url ? url : "https://onedrive.com/files/v-lygi/39192908430" }
+          </div>
+          { actions ? this._renderFooter(actions) : (null) }
+        </div>
+      </div>
+    );
+  }
+
+  private _renderPreviewImage(imageUrl?: string, imageHeight?: string, imageWidth?: string, ogType?: string): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> {
+    const image = (
+      <Image
+        width={ imageWidth }
+        height={ imageHeight }
+        src={ TestImages.documentPreview }
+        role='presentation'
+        alt=''
+      />
+    );
+
+    let icon;
+    switch (ogType) {
+      case "word":
+        icon = <Icon className={ mergeStyles(this._classNames.icon) } iconName='WordDocument' />;
+        break;
+      case "powerpoint":
+        icon = <Icon className={ mergeStyles(this._classNames.icon) } iconName='PowerPointDocument' />;
+        break;
+      case "excel":
+        icon = <Icon className={ mergeStyles(this._classNames.icon) } iconName='ExcelDocument' />;
+        break;
+    }
+
+    return (
+      <div>
+        { image }
+        { icon }
+      </div>
+    );
+  }
+
+  private _renderFooter(actions: IButtonProps[]): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> {
+    return (
+      <div className={ mergeStyles(this._classNames.actions) }>
+        { actions && actions.map((action, index) => {
+          return (
+            <div className={ mergeStyles(this._classNames.action) } key={ index }>
+              <IconButton { ...action } />
+            </div>
+          );
+        }) }
+      </div>
+    );
+  }
+
+  private _onClick = (ev: React.MouseEvent<HTMLElement>): void => {
+    this._onAction(ev);
+  }
+
+  private _onKeyDown = (ev: React.KeyboardEvent<HTMLElement>): void => {
+    if (ev.which === KeyCodes.enter || ev.which === KeyCodes.space) {
```

Comment:
FYI because I fixed a bug elsewhere about this. If the node could potentially be a `<button/>` then this will fire twice. Once for `_onClick` and again for `_onKeyDown` which will in turn invoke `_onAction` twice. Native `onclick` fires for `SPACE` is why this occurs.

Reply:
Good to know, thanks!